### ISSUE TITLE
Semicolon delimited files not working anymore

### ISSUE THREAD

**Issue by christianhans** [wadetb/Sublime-Text-Advanced-CSV#35]
Hi,

before the latest update to v1.1.3 I was able to use "Set delimiter" to set the delimiter to semicolon and then use all of the other actions (such as Justify columns).

Now all the CSV actions that rely on delimiters stopped working.

Best regards

**Comment by wadetb** [comment]
Hi @christianhans, the behavior has changed slightly in the latest version. You may need to re-apply the delimiter setting for any views you have open.

You can use Tools -> CSV -> Set delimiter to set the delimiter character, and that will be remembered with the current view, even across Sublime Text restarts.

Or, you can save the file with a custom extension (ssv?) and add a new delimiter mapping, or even change the default:

```json
{
  "delimiter_mapping": {
  	"*.csv": ",",
  	"*.psv": "|",
  	"*.tsv": "\t",
  	"*.ssv": ";" <<<< add this line
  },
  "delimiter": ",", <<<< or change the default here
  "auto_quote": true
}
```

**Comment by christianhans** [comment]
Hi wadetb,

Thank you for the quick response.

I am using "Set delimiter" to set it to semicolon and my current settings file AdvancedCSV.sublime-settings looks like this:

```
{
  "word_wrap": false,
  "delimiter": ";"
}
```

Here is what I discovered: Whenever I open a new view (File > New file), paste some semicolon-delimited data and set the file type to Advanced CSV, setting delimiter to semicolon is recognized and all actions (such as justify columns) work properly. But as soon as I save the view to a *.csv file, the delimiter setting is automatically set back to comma and I can not even change it manually back to semicolon.

Best regards,
Christian

**Comment by wadetb** [comment]
Thanks for the repro steps, I am able to get the same behavior.

**Comment by wadetb** [comment]
Fixed in 1.1.4. Thanks again for the report!

**Comment by christianhans** [comment]
Can confirm it works now as expected. Thank you @wadetb!
